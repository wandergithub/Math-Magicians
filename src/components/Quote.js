import Header from './Header';

const Quote = () => (
  <div>
    <Header />
    <p>Quote CalcPage</p>
  </div>
);

export default Quote;
