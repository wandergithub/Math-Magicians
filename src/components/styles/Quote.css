.p {  
  font-size: 2em;
  margin: 183px;
}