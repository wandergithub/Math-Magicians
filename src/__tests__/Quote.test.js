import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Quote from '../components/Quote'

describe("Quote page tests", () => {
  it("Renders correctly", () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <Quote />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
