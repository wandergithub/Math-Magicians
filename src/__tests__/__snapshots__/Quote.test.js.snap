// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote page tests Renders correctly 1`] = `
<div>
  <header
    className="nav-bar"
  >
    <h1>
      Math Magicians
    </h1>
    <div
      className="link-container"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        className="mid-b"
        href="/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      <a
        href="/Quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </div>
  </header>
  <p
    className="p"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -William Paul Thurston
  </p>
</div>
`;
